charts/datafold/charts/nginx/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nginx
+  name: nginx
+spec:
+  ports:
+  - name: 80-80
```

Comment:
I would add a port in the deployment and name it:

          ports:
            - name: server
              containerPort: {{ .Values.global.server.port }}
              protocol: TCP
              
 then in service, you can refer to the named port (of the container). You don't need a name in service then:
 
```
  ports:
    - port: {{ include "datafold.server.port" . }}
      targetPort: server
      protocol: TCP
```